### PR TITLE
Need to find andcc equivalent in altera, removed underscore_ in front…

### PR DESCRIPTION
**Pull request by jklasdfj**
… of asm directive, removed reference to memory.h because altera ide doesn't like it, updated to use NULL correctly (capitals), need to fix use of string insertion, updated queue.h to reference qcore.h to remove complaints about Process struct

### CODE REVIEW

File: qmain.c [hunk 1]
```diff
@@ -1,50 +1,49 @@
...
-******************************************************************************/
-
-#include "qinit.h"
-#include "qcore.h"
-#include "qrequest.h"
-#include "quser.h"
-
-/*----------------------------------------------------------------*/
-
-#define cli()	_asm("andcc #$EF\n")	/* clear Interrupt? bit in CCR */ //TODO: change for altera architecture
```
Comment (jklasdfj) [inline qmain.c:20]: Removed the underscore in front of asm. Need to find correct asm command instead of andcc.

File: qrequest.c [hunk 2]
```diff
@@ -1,82 +1,83 @@
...
+		uses accumulator D.
+
+		(C) 1999 by Naraig Manjikian
+		Department of Electrical and Computer Engineering
+		Queen's University
+******************************************************************************/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
```
Comment (jklasdfj) [inline qrequest.c:25]: Added string.h import, commented out memory.h because it's not being recognized.

**Comment by jharon** [comment]
Hello there!

**Comment by jklasdfj** [comment]
I forgot to merge this in, after reviewing it with the team in person.

**Pull request closed without merging**